packages/markdown-it-bi-directional-links/src/index.ts

```diff
@@ -229,12 +235,14 @@ export interface BiDirectionalLinksOptions {
  * @param options.dir - The directory to search for bi-directional links.
  * @param options.baseDir - The base directory joined as href for bi-directional links.
  * @param options.includesPatterns - The glob patterns to search for bi-directional links.
+ * @param options.excludesPatterns - The glob patterns to exclude files from search.
  * @returns A markdown-it plugin.
  */
 export const BiDirectionalLinks: (options?: BiDirectionalLinksOptions) => PluginSimple = (options) => {
   const rootDir = options?.dir ?? cwd()
   const baseDir = options?.baseDir ?? '/'
   const includes = options?.includesPatterns ?? []
+  const excludes = options?.excludesPatterns ?? []
```

Comment:
```suggestion
  const excludes = options?.excludesPatterns ?? ['_*', 'dist', 'node_modules']
```

Or otherwise the user will not be possible to disable the exclude patterns entirely.